src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -600,6 +605,23 @@ public static byte PopCount(int value)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static byte PopCount(ulong value)
         {
+            if (Popcnt.X64.IsSupported)
+            {
+                return (byte)Popcnt.X64.PopCount(value);
+            }
+
+            // TODO: Is it ever the case that X86/X64 is supported but not the other?
+            if (Popcnt.IsSupported)
```

Comment:
`x86` can be supported without `x64` being supported (a 32-bit OS). `x64` support should always imply `x86` support, however.